src/include/session.h

```diff
@@ -301,22 +301,23 @@ struct __wt_session_impl {
 #define WT_SESSION_DEBUG_CHECKPOINT_FAIL_BEFORE_TURTLE_UPDATE 0x000010u
 #define WT_SESSION_DEBUG_DO_NOT_CLEAR_TXN_ID 0x000020u
 #define WT_SESSION_DEBUG_RELEASE_EVICT 0x000040u
-#define WT_SESSION_EVICTION 0x000080u
-#define WT_SESSION_IGNORE_CACHE_SIZE 0x000100u
-#define WT_SESSION_IMPORT 0x000200u
-#define WT_SESSION_IMPORT_REPAIR 0x000400u
-#define WT_SESSION_INTERNAL 0x000800u
-#define WT_SESSION_LOGGING_INMEM 0x001000u
-#define WT_SESSION_NO_DATA_HANDLES 0x002000u
-#define WT_SESSION_NO_RECONCILE 0x004000u
-#define WT_SESSION_PREFETCH_ENABLED 0x008000u
-#define WT_SESSION_PREFETCH_THREAD 0x010000u
-#define WT_SESSION_QUIET_CORRUPT_FILE 0x020000u
-#define WT_SESSION_READ_WONT_NEED 0x040000u
-#define WT_SESSION_RESOLVING_TXN 0x080000u
-#define WT_SESSION_ROLLBACK_TO_STABLE 0x100000u
-#define WT_SESSION_SAVE_ERRORS 0x200000u
-#define WT_SESSION_SCHEMA_TXN 0x400000u
+#define WT_SESSION_DUMP_EXTLIST 0x000080u
```

Comment:
Just a nitpick: would a name like "WT_SESSION_DUMP**ING**_EXTLIST" or "WT_SESSION_EXTLIST_DUMP**ING**" emphasize the purpose of the flag better?

Reply:
yep I agree with `WT_SESSION_DUMPING_EXTLIST`

Reply:
Changed https://github.com/wiredtiger/wiredtiger/pull/12071/commits/2a50d1bc09b08eab2576f51d573a29f7c08756c2